flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -80,19 +80,21 @@
 	 * JVM heap size for the JobManager with memory size.
 	 */
 	@Documentation.CommonOption(position = Documentation.CommonOption.POSITION_MEMORY)
-	public static final ConfigOption<String> JOB_MANAGER_HEAP_MEMORY =
+	public static final ConfigOption<MemorySize> JOB_MANAGER_HEAP_MEMORY =
```

Comment:
Since we should keep the current config option. What's the new name the config option do you prefer?

Reply:
@dawidwys Opinions? How about `JOB_MASTER_HEAP_MEMORY`?